functions/scheduled.js

```diff
@@ -193,11 +257,6 @@ exports.aggregateUsersLocationsInVenue = functions.pubsub
         });
       }
 
-      batch.commit().catch((error) => {
-        throw new HttpsError(
-          "internal",
-          `Commit batch of recent users of venues failed. Error: ${error}`
-        );
-      });
+      return batch.commit();
     });
```

Comment:
Please, leave the error there. It can be useful

Reply:
Since this is an `asyn` function anything you `await` or return from it will be handled by an functions executor and all errors will show up in logs like this:
<img width="1177" alt="image" src="https://user-images.githubusercontent.com/16973041/135394075-07e4c2ef-e2b0-49dc-b3ff-e298294620c3.png">
Maybe I am missing something, but I don't see a point of catching and rethrowing here.

Reply:
Sounds reasonable